### ISSUE TITLE
Own statistic values in one line.

### ISSUE THREAD

**Issue by 2Abendsegler** [2Abendsegler/GClh#800]
Own statistic values in one line.
https://www.geocaching.com/my/statistics.aspx

![x1](https://user-images.githubusercontent.com/22332216/44709458-ffa50580-aaa9-11e8-9843-bfca26637f5d.JPG)

**Comment by 2Abendsegler** [comment]
![x2](https://user-images.githubusercontent.com/22332216/44709562-37ac4880-aaaa-11e8-8614-49f1f05b1cf4.JPG)

**Comment by Ruko2010** [comment]
@2Abendsegler Ich dachte du hast Urlaub 😃

**Comment by 2Abendsegler** [comment]
Ja, aber die Sachen nerven mich schon lange und ich hatte gerade noch Zeit. :)
